docs/developer/chart-repo.md

```diff
@@ -0,0 +1,77 @@
+# Kubeapps chart-repo Developer Guide
+
+The `chart-repo` component is tool that scans a Helm chart repository and populates chart metadata in a MongoDB server. This metadata is then served by the `chartsvc` component.
+
+## Prerequisites
+
+- [Git](https://git-scm.com/)
+- [Docker CE](https://www.docker.com/community-edition)
+- [Kubernetes cluster](https://kubernetes.io/docs/setup/pick-right-solution/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Telepresence](https://telepresence.io)
```

Comment:
is telepresence really needed? We can say that it's just a recommendation for an easier development setup.

Reply:
hmm.. not really needed that's true

Reply:
added a small comment about telepresence